tests/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_C_COMPILER "$ENV{HOME}/cross/bin/aarch64-w64-mingw32-gcc")
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_COMPILER "$ENV{HOME}/cross/bin/aarch64-w64-mingw32-c++")
+set(CMAKE_RANLIB "$ENV{HOME}/cross/bin/aarch64-w64-mingw32-ranlib")
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -fopenmp")
+
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/b10fad38c4026a29ea6561ab15fc4818170d1c10.zip
+)
+
+FetchContent_MakeAvailable(googletest)
+
+cmake_minimum_required(VERSION 3.14)
+project(aarch64_mingw_tests)
+
+enable_testing()
+
+add_executable(
+  aarch64_mingw_tests
```

Comment:
This creates executable without the .exe extension. It needs to be renamed manually to execute it.

Reply:
yeah, I was thinking about it and I was not sure what will be right extension for executable. it will be changed.

Reply:
Done